keyboards/splitkb/kyria/keymaps/default/readme.md

```diff
@@ -0,0 +1,258 @@
+<h1 align="center"> Kyria's Default Keymap</h1>
```

Comment:
Don't use HTML here.

Reply:
Can I keep the HTML formatting for the figure and KLE link or should that formatting be stripped too?

Reply:
Ideally, all HTML should be removed.

Reply:
Solved in https://github.com/qmk/qmk_firmware/pull/14080/commits/dd1505e6c3dd74bfb6174ab602c9ace0b56f4750